inspire/base/templates/search/search_base.html

```diff
@@ -0,0 +1,182 @@
+{#
+# This file is part of INSPIRE.
+# Copyright (C) 2015 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% extends "search/searchbar_frame.html" %}
+
+{% from "search/results.html" import render_search_results with context %}
+
+{% block global_bundles %}
+  {{ super() }}
+  {% bundles "formatter.css" %}
+{% endblock %}
+
+{%- set portalboxes = dict() -%}
+{%- for k,l in collection.portalboxes_ln|groupby('position') -%}
+  {%- do portalboxes.update({k:l}) -%}
+{%- endfor -%}
+
+{% block title %}
+  {{ portalboxes.tp }}
+  {{ super() }}
+  {{ portalboxes.te }}
+{% endblock %}
+{% block header %}
+  {{ super() }}
+  {%- set args = request.args.copy().to_dict() -%}
+  {%- set jrec = (pagination.page-2)*pagination.per_page+1 if pagination.has_prev else 1 -%}
+  {%- do args.update({'jrec': jrec}) -%}
+  {%- if pagination.has_prev -%}
+  <link rel="prev" href="{{ url_for(request.endpoint, **args) }}"/>
+  {%- endif -%}
+  {%- if pagination.has_next -%}
+  {%- set jrec = (pagination.page)*pagination.per_page+1 if pagination.has_next else (pagination.pages-1)*pagination.per_page+1 -%}
+  {%- do args.update({'jrec': jrec}) -%}
+  <link rel="next" href="{{ url_for(request.endpoint, **args) }}"/>
+  {%- endif -%}
+{% endblock %}
+
+{%- block inner_content -%}
+  {%- block index -%}
+    {%- if not records|length -%}
+      {%- block index_noresult -%}
+        <div class="row">
+          <div class="col-md-12">
+            <p>
+              <strong>{{ _('Your search did not match any records. Please try again.') }}</strong>
+            </p>
+            {{ create_nearest_terms_box()|safe }}
+          </div>
+        </div>
+      {%- endblock index_noresult -%}
+    {%- else -%}
+      {%- block index_results -%}
+        <div class="row">
+          {%- block search_bar %}
```

Comment:
If the rest of the template is exactly the same then just having in your template `extends` and:

```
{%- block search_bar %}
            <div id="collection-search-bar" class="col-md-12">
              <div id="search-bar">
                <form name="search" action="/search">
                  <div class="input-group">
                    <input type="text" name="p" placeholder="Search" autofocus>
                    <input type="hidden" name="cc" value="{{ collection.name | lower }}">
                      <span class="input-group-btn" id="{{ collection.name | lower }}-search">
                      <button class="btn btn-primary btn-block">
                        <i class="fa fa-search fa-2x"></i>
                      </span>
                      </button>
                  </div>
                </form>
              </div>
            </div>
          {%- endblock search_bar %}
```

should work.